### PR TITLE
Week 1 Assignment

### PR DESCRIPTION
**Pull request by trungngth**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+)
+
+func main() {
+	fmt.Print("> ")
+	scanner := bufio.NewScanner(os.Stdin)
+	for scanner.Scan() {
+		input := scanner.Text()
+		element := strings.Split(input, " ")
+		if len(element) != 3 {
+			fmt.Printf("Missing arguments or way too many arguments! %s\n", element)
```
Comment (xuanit) [inline main.go]: Fyi: https://golang.org/pkg/log/#Fatal

File: main.go [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+	switch operand {
+	case "+":
+		result = float64(firstNumber) + float64(secondNumber)
+	case "-":
+		result = float64(firstNumber) - float64(secondNumber)
+	case "*":
+		result = float64(firstNumber) * float64(secondNumber)
+	case "/":
+		result = float64(firstNumber) / float64(secondNumber)
```
Comment (xuanit) [inline main.go:47]: what if `secondNumber` is zero.

File: main.go (hunk 1 above)
Comment (trungngth) [reply main.go]: Thank you. This is very useful.

File: main.go (hunk 2 above)
Comment (trungngth) [reply main.go:47]: I see. I will fix this in the next commit ^^

**Pull request merged**